src/tests/tests.rs

```diff
@@ -0,0 +1,14 @@
+use super::Histogram;
+
+#[path = "helpers.rs"]
```

Comment:
Thanks for being flexible! I shall also try to be slightly more granular than I normally would be in my PR's. ;) 

This sort of stuff is all so path-dependent: once bitten twice shy, and where we've been bitten is very individual.